src/main.ts

```diff
@@ -2,10 +2,11 @@ import { NestFactory } from "@nestjs/core";
 import { AppModule } from "./app.module";
 import { SwaggerModule, DocumentBuilder } from "@nestjs/swagger";
 import { RequestMethod } from "@nestjs/common";
-
+import { join } from "path";
+import express = require("express");
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
-
+  app.use("/uploads", express.static(join(__dirname, "..", "uploads")));
```

Comment:
Don't hardcode the path, use it from an environment variable.

Reply:
ok sir